toolkits/springer/packages/springer-listing/demo/context.json

```diff
@@ -34,14 +35,16 @@
 					"Original Paper",
 					"4 October 2019"
 				],
+				"content": "Morbi luctus eleifend velit ut sodales. Donec semper mauris nec tristique semper. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Morbi mi diam, commodo pharetra pharetra quis, tempus ac felis. Suspendisse at felis ut lorem mollis tristique feugiat eu turpis. Phasellus fringilla lacus ac libero tempus fringilla. Suspendisse sit amet risus dignissim, varius nunc vitae, pellentesque risus. Phasellus tempor dui metus, in mollis sem mollis feugiat. Cras feugiat facilisis quam eu vulputate. Integer ullamcorper arcu in tortor dictum sagittis. Mauris quis leo at nulla bibendum porta non quis ante.",
 				"image": {
-				"imageSrc": "https://picsum.photos/800/700",
-				"imageAlt": "Descriptive alternative text"
+					"imageSrc": "https://picsum.photos/800/700",
```

Comment:
would we need to remove this little additional whitespace that's found it's way in? Or do we actually need to give it a full tabs width as it's 'inside' the `"image"`?

Reply:
Yeah, I need to neaten this up.